src/setup-beam.js

```diff
@@ -49,7 +50,7 @@ async function main() {
   await maybeInstallRebar3(rebar3Spec)
 
   // undefined is replaced by a function, post- main branch merge
-  const setupBeamVersion = '6c8deb6'
+  const setupBeamVersion = 'b80b314'
```

Comment:
It's strange you'd do this change, so it's probably coming from not using the most recent `main` branch. You should probably rebase.